### ISSUE TITLE
Status of this Driver?

### ISSUE THREAD

**Issue by smartclash** [botman/driver-discord#3]
Well, Discord has grown a lot these days. And, the only PHP library that is actively developed for Discord bot is this. But, in the botman.io website, I don't see anything about the Discord driver. So, my mind started picking on me.

Is this Driver still alive, or just died

**Comment by mpociot** [comment]
The problem with the Discord driver is that we use the discord-php package underneath which is no longer under active development: https://github.com/teamreflex/DiscordPHP

We could fork and maintain this package, but I do not really want to do this.

**Comment by smartclash** [comment]
@mpociot, yup. You are right. So this means that this driver is also stopped unless any other good package is not found. Right?

**Comment by robjbrain** [comment]
@mpociot I'd be interesting in creating a package for discord (I need one for my current project) but i've not located a guide to creating drivers? Is there one or is it just a case of looking at existing drivers as an example? Is this package a good example and it is simply that DiscordPHP needs to be replaced?

**Comment by janvernieuwe** [comment]
@robjbrain I was also looking for this driver, I'll watch this topic in case you start a repo!

**Comment by FrostBy** [comment]
@mpociot 
What about https://github.com/CharlotteDunois/Yasmin ?

**Comment by gtjamesa** [comment]
Is there any update on this?

**Comment by re-thc** [comment]
@FrostBy Yasmin is still in active development so is a good choice. Hope it gets implemented.

**Comment by kvarnagit** [comment]
Any new status here?

**Comment by JABirchall** [comment]
I have a working discord driver here: https://github.com/JABirchall/botman-driver-discord using Yasmin.

**Comment by kocoten1992** [comment]
https://github.com/teamreflex/DiscordPHP/graphs/contributors seem to be alive again :100:

**Comment by PHLAK** [comment]
Any chance this can be revived? If so, what needs to be done to get this up and running? I'd like to help out if possible.

Maybe I can take a crack at updating the `teamreflex/DiscordPHP` dependency to `discord-php/DiscordPHP`.

**Comment by crustamet** [comment]
> Any chance this can be revived? If so, what needs to be done to get this up and running? I'd like to help out if possible.
> 
> Maybe I can take a crack at updating the `teamreflex/DiscordPHP` dependency to `discord-php/DiscordPHP`.

this would be amazing srsly..
